packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -83,6 +83,10 @@ const PersonalDetails = ({
         }
 
         errors.document_number = isDocumentNumberValid(document_number, document_type);
+
+        if (document_type.id !== IDV_NOT_APPLICABLE_OPTION.id && !values.confirmation_checkbox) {
+            errors.confirmation_checkbox = 'Hit the checkbox';
```

Comment:
shouldnt we localize this error?

Reply:
I changed it to just 'error'
this text is not visible, just needed for correct handling formik status